tests/api/common/test_airflow_health.py

```diff
@@ -33,8 +33,12 @@
 @patch("airflow.api.common.airflow_health.SchedulerJobRunner.most_recent_job", return_value=None)
 @patch("airflow.api.common.airflow_health.TriggererJobRunner.most_recent_job", return_value=None)
 @patch("airflow.api.common.airflow_health.DagProcessorJobRunner.most_recent_job", return_value=None)
+@patch("airflow.api.common.airflow_health.conf.getboolean", return_value=True)
```

Comment:
We should mock the appropriate conf option `standalone_dag_processor` instead of the whole `getboolean` method. That could incure undesired side effect for other conf options.

`@conf_vars(...)`

Reply:
Done